### PR TITLE
Unignore the upload from file with reporter test

### PR DESCRIPTION
**Pull request by gapra-msft**
Resolves https://github.com/Azure/azure-sdk-for-java/issues/9189

### CODE REVIEW

File: sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/FileAPITest.groovy [hunk 1]
```diff
@@ -2233,9 +2233,8 @@ class FileAPITest extends APISpec {
         StepVerifier.create(fac.uploadFromFile(file.toPath().toString(), parallelTransferOptions,
             null, null, null))
             .verifyComplete()
-
-        // Check if the reported size is equal to or greater than the file size in case there are retries.
-        uploadReporter.getReportedByteCount() >= size
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/FileAPITest.groovy:2238]: yikes

**Review by kasobol-msft — APPROVED** [review]

**Comment by gapra-msft** [comment]
Live test failures are due to a known issue with OAuth tests

**Pull request merged**